examples/vsphere/sample-terraform-dhcp.tfvars

```diff
@@ -156,22 +158,6 @@ node_pools = {
     node_labels = {
       "workload.sas.com/class" = "stateless"
     }
-  },
```

Comment:
this block is pretty important when it comes to running singlestore on our open sourced setup. I don't see this code in its own example and or placed anywhere else.

Reply:
Given the need to pre-provision nodes for single store on our upstream Kubernetes here, that is needed. The other providers can dynamically setup disk and items for Single Store when they request the resources in the cloud providers. Bare-metal and VMWare cannot.